cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -150,8 +155,9 @@ public ClusterOperatorConfig(
             int zkAdminSessionTimeoutMs,
             int dnsCacheTtlSec,
             boolean podSetReconciliationOnly,
-            int podSetControllerWorkQueueSize) {
-        this.namespaces = unmodifiableSet(new HashSet<>(namespaces));
+            int podSetControllerWorkQueueSize,
+            String operatorName) {
+        this.namespaces = Set.copyOf(namespaces);
```

Comment:
Why was this change needed?

Reply:
Accomplishes the same thing with one less new object created, and it's simpler to read.